examples/worldview/src/features/map/DebugOverlay.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+/* eslint-disable complexity */
+export function DebugOverlay({containerRef, deckRef, dimension, previewSize, dpi, onDpiChange}) {
+  const container = containerRef && containerRef.current;
```

Comment:
would it be better to use optional chaining? ``` const container = containerRef?.current;```
same with other variables till line 9

Reply:
Yeah, I just haven't adopted it yet since last I checked it was "draft" syntax. Maybe it's official now?